src/foam/nanos/notification/email/SMTPEmailService.js

```diff
@@ -243,160 +205,49 @@ return config_;`
           type: 'foam.nanos.notification.email.EmailMessage'
         }
       ],
-      javaCode: `
-if ( ! this.getEnabled() ) return;
-
-((FixedThreadPool) getThreadPool()).submit(x, new ContextAgent() {
-  @Override
-  public void execute(X x) {
-    try {
-      MimeMessage message = createMimeMessage(finalizeEmailConfig(x, emailMessage));
-      if ( message == null ) {
-        return;
-      }
-
-      // send message
-      Transport transport = session_.getTransport("smtp");
-      transport.connect();
-      transport.sendMessage(message, message.getAllRecipients());
-      transport.close();
-    } catch (Throwable t) {
-      t.printStackTrace();
-    }
-  }
-});`
-    },
-    {
-      name: 'sendEmailFromTemplate',
-      javaCode: `
-if ( ! this.getEnabled() ) return;
-
-String group = user != null ? (String) user.getGroup() : null;
-EmailTemplate emailTemplate = DAOResourceLoader.findTemplate(getX(), name, group);
-if ( emailMessage == null )
-  return;
-
-for ( String key : templateArgs.keySet() ) {
-  Object value = templateArgs.get(key);
-  if ( value instanceof String ) {
-    String s = (String) value;
-    templateArgs.put(key, new String(s.getBytes(StandardCharsets.ISO_8859_1), StandardCharsets.UTF_8));
-  }
-}
-
-EnvironmentConfiguration config = getConfig(group);
-JtwigModel model = JtwigModel.newModel(templateArgs);
-emailMessage = (EmailMessage) emailMessage.fclone();
-
-JtwigTemplate templateBody =    JtwigTemplate.inlineTemplate(emailTemplate.getBody(), config);
-emailMessage.setBody(templateBody.render(model));
-
-// If subject has already provided, then we don't want to use template subject.
-if (SafetyUtil.isEmpty(emailMessage.getSubject())) {
-  JtwigTemplate templateSubject = JtwigTemplate.inlineTemplate(emailTemplate.getSubject(), config);
-  emailMessage.setSubject(templateSubject.render(model));
-}
+      javaCode:
+        `
+        if ( ! this.getEnabled() ) return;
```

Comment:
Why even have this flag?

Reply:
On the nanopay side we are overriding the email(service) with the smtp implementation. If this is to remain in the build, we would want this off by default.

Will talk to you in person

Reply:
New ticket to encompass a future fix: https://nanopay.atlassian.net/browse/CPF-870